From: Jochen Sprickerhof <jspricke@debian.org>
Date: Tue, 31 Oct 2023 17:30:17 +0100
Subject: Fix undefined style include

The css file is already included in css_files below.
---
 site/source/_themes/emscripten_sphinx_rtd_theme/layout.html | 5 -----
 1 file changed, 5 deletions(-)

--- a/site/source/_themes/emscripten_sphinx_rtd_theme/layout.html
+++ b/site/source/_themes/emscripten_sphinx_rtd_theme/layout.html
@@ -33,11 +33,6 @@
 
   {% endif %}
 
-  {# RTD hosts this file, so just load on non RTD builds #}
-  {% if not READTHEDOCS %}
-    <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
-  {% endif %}
-
   {% for cssfile in css_files %}
     <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
   {% endfor %}
